run.py

```diff
@@ -0,0 +1,112 @@
+"""Main runnable file for the codebase"""
+
+import argparse
+import logging
+import asyncio
+from multiprocessing import Queue
+from flight import logger, upload_mission, run_mission
+import flight.config as config
+from mavsdk import System
+from flight.flight import wait_for_drone, log_flight_mode, observe_is_in_air, DroneNotFoundError
+
+SIM_ADDR: str = "udp://:14540"  # Address to connect to the simulator
+CONTROLLER_ADDR: str = "serial:///dev/ttyUSB0"  # Address to connect to a pixhawk board
+
+
+async def init_and_begin(simulation: bool, competition: bool) -> None:
+    """
+    Creates drone object and passes it to start_flight
+
+    Parameters
+    ----------
+    simulation: bool
+        Decides which connection address to use
```

Comment:
Be a bit clearer here. Something like "When true, run with simulation connection address, otherwise..."